engine/uk.co.spudsoft.birt.emitters.excel/src/uk/co/spudsoft/birt/emitters/excel/ExcelEmitter.java

```diff
@@ -408,7 +408,11 @@ public void end(IReportContent report) throws BirtException {
 				}
 			}
 			if (handlerState.getWb() instanceof SXSSFWorkbook) {
-				((SXSSFWorkbook) handlerState.getWb()).dispose();
+				try {
+					((SXSSFWorkbook) handlerState.getWb()).close();
+				} catch (IOException ex) {
+					log.debug("ex:", ex.toString());
```

Comment:
If there is an exception, I thought it's better to call a log method that shows the stack trace of the exception.